_data/entertainment.yml

```diff
@@ -99,6 +99,8 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
+      exceptions:
+          text: "Software implementation requires use of the Authy service."
       doc: http://blog.twitch.tv/2015/11/two-factor-authentication-now-available-on-your-twitch-account/
```

Comment:
Their doc is now available over `HTTPS`. Would you like to add that change to your PR?